users/users_test.go

```diff
@@ -72,13 +72,6 @@ func TestValidate(t *testing.T) {
 			},
 			err: users.ErrMalformedEntity,
 		},
-		"validate user with empty password": {
-			user: users.User{
-				Email:    email,
-				Password: "",
-			},
-			err: users.ErrMalformedEntity,
-		},
```

Comment:
There is a reason to remove this test?

Reply:
Removed because password validation is removed from users/api/requests.go